Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -536,6 +536,8 @@ metsRightsOwnerLogo=METS Rechteinhaber Logo
 metsRightsOwnerMail=METS Rechteinhaber Kontakt
 metsRightsOwnerSite=METS Rechteinhaber URL
 metsfilegroups=Unterordner und METS Dateigruppen
+migrateProjects=Migrieren
```

Comment:
Please use this key in `system.xhtml` as tab title (instead of `indexing`, which is likely a copy & paste error)